.gitignore

```diff
@@ -13,3 +13,5 @@ node_modules
 dist
 docs/docs
 .DS_Store
+*.swp
+cscope.*
```

Comment:
I dunno if this is necessary, let's just move forward with this patch.

Rebase is already going to be hell. :)